junction/templates/proposals/detail/base.html

```diff
@@ -178,4 +178,7 @@
     });
 
 </script>
+{% block comment_vote_script %}
+    
+{% endblock comment_vote_script %}
```

Comment:
You can add your script at the bottom of this block. Don't override it.


Thanks & Regards,
Sivasubramaniam Arunachalam

On Thu, Jan 8, 2015 at 11:42 PM, nabeel07 <notifications@github.com> wrote:

> In junction/templates/proposals/detail/base.html
> <https://github.com/pythonindia/junction/pull/46#discussion-diff-22669039>
> :
>
> > @@ -178,4 +178,7 @@
> >      });
> >
> >  </script>
> > +{% block comment_vote_script %}
> > +
> > +{% endblock comment_vote_script %}
>
> It affects the "ProposalVoting" section.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/pythonindia/junction/pull/46/files#r22669039>.
>